<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qMRMLTableViewControllerWidget</class>
 <widget class="ctkPopupWidget" name="qMRMLTableViewControllerWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>472</width>
    <height>367</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Table View Controller</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinimumSize</enum>
     </property>
     <item>
      <widget class="qMRMLNodeComboBox" name="tableComboBox">
       <property name="nodeTypes">
        <stringlist>
         <string>vtkMRMLTableNode</string>
        </stringlist>
       </property>
       <property name="noneEnabled">
        <bool>true</bool>
       </property>
       <property name="addEnabled">
        <bool>true</bool>
       </property>
       <property name="removeEnabled">
        <bool>true</bool>
       </property>
       <property name="editEnabled">
        <bool>true</bool>
       </property>
       <property name="renameEnabled">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="LockTableButton">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>30</width>
         <height>0</height>
        </size>
       </property>
       <property name="toolTip">
        <string>Allow table editing</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
         <normaloff>:/Icons/Medium/SlicerUnlock.png</normaloff>:/Icons/Medium/SlicerUnlock.png</iconset>
       </property>
       <property name="checkable">
        <bool>false</bool>
       </property>
       <property name="flat">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QToolButton" name="CopyButton">
       <property name="toolTip">
        <string>Copy</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
         <normaloff>:/Icons/Medium/SlicerEditCopy.png</normaloff>:/Icons/Medium/SlicerEditCopy.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="PasteButton">
       <property name="toolTip">
        <string>Paste</string>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
         <normaloff>:/Icons/Medium/SlicerEditPaste.png</normaloff>:/Icons/Medium/SlicerEditPaste.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="PlotButton">
       <property name="toolTip">
        <string>Generate an Interactive Plot based on user-selection of the columns of the table.</string>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset resource="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc">
         <normaloff>:/Icons/Medium/SlicerInteractivePlotting.png</normaloff>:/Icons/Medium/SlicerInteractivePlotting.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="EditControlsFrame">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>15</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Plain</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::MinimumExpanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>16</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="ColumnInsertButton">
          <property name="toolTip">
           <string>Add column</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/column_add.png</normaloff>:/Icons/column_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="ColumnDeleteButton">
          <property name="toolTip">
           <string>Delete column</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/column_delete.png</normaloff>:/Icons/column_delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="LockFirstColumnButton">
          <property name="toolTip">
           <string>Lock first column</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/row_header_lock.png</normaloff>:/Icons/row_header_lock.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Preferred</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>8</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QToolButton" name="RowInsertButton">
          <property name="toolTip">
           <string>Add row</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/row_add.png</normaloff>:/Icons/row_add.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="RowDeleteButton">
          <property name="toolTip">
           <string>Delete row</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/row_delete.png</normaloff>:/Icons/row_delete.png</iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QToolButton" name="LockFirstRowButton">
          <property name="toolTip">
           <string>Lock first row</string>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="icon">
           <iconset resource="../qMRMLWidgets.qrc">
            <normaloff>:/Icons/column_header_lock.png</normaloff>:/Icons/column_header_lock.png</iconset>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>1</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <spacer name="BottomSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>0</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>qMRMLNodeComboBox</class>
   <extends>QWidget</extends>
   <header>qMRMLNodeComboBox.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ctkPopupWidget</class>
   <extends>QFrame</extends>
   <header>ctkPopupWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../qMRMLWidgets.qrc"/>
  <include location="../../../../../Base/QTGUI/Resources/qSlicerBaseQTGUI.qrc"/>
 </resources>
 <connections/>
</ui>
